### PR TITLE
Increase Flask worker and GKE load balancer timeout to 60s

### PR DESCRIPTION
**Pull request by shifucun**
The default timeout is 30s, which prevents a large stat-var-hierarchy cache to serve.

### CODE REVIEW

File: gke/mcs.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ kind: MultiClusterService
 metadata:
   name: website-mcs
   namespace: website
+  annotations:
+    cloud.google.com/backend-config: '{"ports": {"8080":"backendconfig"}}'
```
Comment (pradh) [inline gke/mcs.yaml:21]: This is supposed to refer to the yaml? worth commenting so?

File: build/Dockerfile [hunk 2]
```diff
@@ -36,4 +36,4 @@ COPY server/. /workspace/server/
 
 # Run server
 WORKDIR /workspace/server
-CMD if [ "$ENV" = "local" ] ; then python3.7 main.py ; else exec gunicorn --bind :8080 main:app ; fi
\ No newline at end of file
+CMD if [ "$ENV" = "local" ] ; then python3.7 main.py ; else exec gunicorn --timeout 60 --bind :8080 main:app ; fi
```
Comment (pradh) [inline build/Dockerfile:39]: for my understanding, is the 60s in the yaml for prod and this one for local/dev?

**Review by pradh — APPROVED** [review]

File: gke/mcs.yaml (hunk 1 above)
Comment (shifucun) [reply gke/mcs.yaml:23]: Done

File: build/Dockerfile (hunk 2 above)
Comment (shifucun) [reply build/Dockerfile:39]: This is for Kubernetes deployment, ie when "ENV" != "local". So this applies to prod/dev

Local instance will use "python3.7 main.py" which does not have a timeout.

**Review by shifucun — COMMENTED** [review]
Thanks for review!

**Pull request merged**